programs/drift/src/controller/insurance.rs

```diff
@@ -569,6 +569,7 @@ pub fn settle_revenue_to_insurance_fund(
     }
 
     if spot_market.insurance_fund.user_shares > 0 {
+        // only allow MAX_APR_PER_REVENUE_SETTLE_TO_INSURANCE_FUND_VAULT or half revenue pool to be settled
```

Comment:
`token_amount.safe_div(5)?` is 20%?

Reply:
yes, so only up to 20% of remaining revenue pool can be settled to IF in one settle